static/app/components/events/eventReplay/replayPreviewPlayer.tsx

```diff
@@ -224,4 +221,12 @@ const StyledReplayCell = styled(ReplayCell)`
   padding: 0 0 ${space(1)};
 `;
 
+const HeaderWrapper = styled('div')`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: ${space(1)};
+  margin-bottom: ${space(1)};
```

Comment:
is this margin-bottom needed?

Reply:
don't need the `gap` here either

Reply:
@ryan953 don't need the gap but the margin bottom makes sure we don't have the button too close to the actual replay player

Reply:
the username is far from the player too now. maybe like `align-items: top` or whatever is what it needs